test/suite/compact_util.py

```diff
@@ -44,9 +44,6 @@ def delete_range(self, uri, num_keys):
     def get_bg_compaction_running(self):
         return self.get_stat(stat.conn.background_compact_running)
 
-    def get_bg_compaction_success(self):
```

Comment:
Does it mean we are no longer testing this stat?

Reply:
they are still used in compact10 and compact11

Reply:
So how is it safe to remove it from `compact_util`?